dist/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <title>insert_site_name</title>
+    <meta name="description" content="insert_site_description">
+    
+    
+    <meta property="fb:app_id" content="1612424139009027" />
+    <meta property="og:site_name" content="Major League Hacking">
+    <meta property="og:title" content="insert_site_name">
+    <meta property="og:description" content="insert_site_description">
+    <meta property="og:type" content="website">
+    
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@MLHacks">
+    <meta name="twitter:creator" content="@MLHacks">
+    <meta name="twitter:title" content="insert_site_name">
+    <meta name="twitter:description" content="insert_site_description">
+    
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700,300italic,400italic,700italic' rel='stylesheet' type='text/css'>
+  <link href="boilerplate.v1.0.0.min.css" rel="stylesheet"></head>
```

Comment:
Looks like this file may need to be regenerated as it's still showing boilerplate instead of mlh